packages/markdown-docx/src/ToCiceroMarkVisitor.js

```diff
@@ -103,6 +103,11 @@ class ToCiceroMarkVisitor {
                 elementType: nodeInformation.elementType,
                 name: nodeInformation.name,
             };
+        } else if (nodeInformation.nodeType === 'Code') {
```

Comment:
Instead of creating another constant `'Code'`, I think we can use the attributes under `definedNodes` only. You would have to change it for all entities so make a separate PR for it.

While you're at it, change `definedNodes` to `DEFINED_NODES` as JS suggests a format for naming constants.

Reply:
@algomaster99 
Will create a separate PR after this PR gets merged.

Reply:
Sure